.travis.yml

```diff
@@ -1,9 +1,19 @@
-language: cpp
-sudo: false
+language: python
+python:
+- 3.7-dev
+os: linux
+dist: xenial
```

Comment:
change this to 18.04

Reply:
Travis still does not support Latest ubuntu.
But this is only the platform for running the docker. Not the build itself.
The build platform version is defined in Blockchain-ops repo.